templates/main.php

```diff
@@ -1187,7 +1187,7 @@ class="otpSecret"/></label>
 
 <div id="encryptionKeyDialog" style="display: none;">
   <p><?php p($l->t('Enter your encryption key.')); ?></p>
-  <p><?php p($l->t('If this if the first time you use Passman, this key will be used for encryption your
+  <p><?php p($l->t('If this is the first time you use PassMan, this key will be used for encryption of your
```

Comment:
Okay, if you want it to be "Passman", then I'll start over again and create a new pull request.